ansible_collections/arista/avd/roles/eos_config_deploy_cvp/tasks/v3/present.yml

```diff
@@ -29,7 +29,7 @@
   register: cvp_container_results
 
 - name: "Execute pending tasks on {{ inventory_hostname }}"
-  tags: [apply]
+  tags: [apply, containers, provision]
```

Comment:
So the `apply` tag is only supposed to be used with `--skip-tags` and not alone with `--tags`. Your change here might make sense for your specific workflow, but it does not match everything else in the role, so it will be even harder for someone else to figure out what to do.
If we want to implement the tags for the apply steps as well, we should do it everywhere, but I am a bit worried about the side-effects, so doing this in a minor release is not good.

Example, someone runs with `--tags provision` today. They get all the changes applied to cloudvision, but no tasks are executed (hitless). If we implement your suggestion, we would suddenly execute those tasks. That is a (dangerous) behavioral change.

Instead we can add some new tags. For this task it could be `execute_cv_tasks_for_container_changes` and for the one above `execute_cv_tasks_for_configlet_changes` and below `execute_cv_tasks_for_device_changes`, giving granular control.
(Better names are welcome, but it needs to be clear - and documented).

Reply:
@ClausHolbechArista - the information you provided here should likely be included in the [documentation](https://avd.arista.com/4.4/roles/eos_config_deploy_cvp/index.html?h=skip+tags#run-module-with-different-tags), making it more prescriptive.

I've reverted the tag-based change for this specific task and tested the variables throughout.